### PR TITLE
feat: split frequency type further

### PR DESCRIPTION
**Pull request by holtgrewe**

### CODE REVIEW

File: frontend/ext/varfish-api/src/lib/types.gen.ts [hunk 1]
```diff
@@ -1985,25 +1957,11 @@ export type SeqvarsQuerySettingsDetails = {
...
-    inhouse_enabled?: boolean;
-    inhouse_carriers?: number | null;
-    inhouse_homozygous?: number | null;
-    inhouse_heterozygous?: number | null;
-    inhouse_hemizygous?: number | null;
+    gnomad_exomes?: unknown;
+    gnomad_genomes?: unknown;
+    gnomad_mitochondrial?: unknown;
+    helixmtdb?: unknown;
+    inhouse?: unknown;
```
Comment (Gregoor) [inline frontend/ext/varfish-api/src/lib/types.gen.ts:1964]: @holtgrewe it looks like type introspection did not work here

Comment (holtgrewe) [reply frontend/ext/varfish-api/src/lib/types.gen.ts:1964]: Sorry @Gregoor Will fix on Wednesday. Will be the equivalent of

https://github.com/varfish-org/varfish-server/blob/3c325fe892edde0a37da9030418392018439c93b/backend/seqvars/models.py#L21

Comment (holtgrewe) [reply frontend/ext/varfish-api/src/lib/types.gen.ts:1964]: Should work after https://github.com/varfish-org/varfish-server/pull/1786

Comment (Gregoor) [reply frontend/ext/varfish-api/src/lib/types.gen.ts:1964]: Thank you!

**Pull request merged**